sql-statements/sql-statement-backup.md

```diff
@@ -112,17 +112,7 @@ BR supports backing up data to S3 or GCS:
 BACKUP DATABASE `test` TO 's3://example-bucket-2020/backup-05/?access-key={YOUR_ACCESS_KEY}&secret-access-key={YOUR_SECRET_KEY}';
 ```
 
-<CustomContent platform="tidb">
-
-The URL syntax is further explained in [external storage URI](/br/backup-and-restore-storages.md#uri-format).
-
-</CustomContent>
-
-<CustomContent platform="tidb-cloud">
-
-The URL syntax is further explained in [external storage URI](https://docs.pingcap.com/tidb/stable/backup-and-restore-storages#uri-format).
-
-</CustomContent>
+The URI syntax is further explained in [URI Formats of External Storage Services](/external-storage-uri.md).
```

Comment:
As `sql-statements/sql-statement-import-into.md` is also used by TiDB Cloud TOC, please use the CustomContent tag for this paragraph.